ChatQnA/docker/xeon/README.md

```diff
@@ -234,7 +234,7 @@ docker compose -f compose.yaml up -d
 If use vLLM backend.
 
 ```bash
-docker compose -f compose_vllm.yaml up -d
+docker compose -f docker_compose_vllm.yaml up -d
```

Comment:
The path has been unified, 
![Uploading image.png…]()

Reply:
yes, update for new file name